dluhc-component-library/src/components/timetablePage/AccordionDropdown.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import { IndicationArrow } from "../inlinesvgs/IndicationArrow";
+
+const AccordionDropdown = ({ information }: { information?: string }) => {
+  const [isClicked, setClicked] = useState(false);
+
+  const toggle = () => {
+    isClicked == true ? setClicked(false) : setClicked(true);
+  };
+
+  return (
+    <>
+      <div className="pt-1 pb-3">
+        <IndicationArrow
+          className={
+            "h-3 w-3 transition-all inline-block" +
+            (isClicked ? " rotate-90" : "")
```

Comment:
Can we please pull this optional class out into a variable and use string interpolation here, it will make it easier to read.

`const rotationClass = isClicked ? "rotate-90" : "";`

`className={`h-3 w-3 transition-all inline-block ${rotationClass}`}